resources/letters/models/letter_generated_base.yml

```diff
@@ -29,3 +29,8 @@ allOf:
         enum:
           - letter
         default: letter
+
+      campaign_id:
```

Comment:
nitpick: could you put this above the object property? that generally should be the last one listed